src/Data/Binary/Parsing.hs

```diff
@@ -0,0 +1,60 @@
+-- | Simple parsing utilities.
```

Comment:
I feel like I commented on something like this some time ago :thinking: but I think this should be `Binary.Decoding`, as "parsing" is used more in the context of language.

Reply:
90b0c96